src/main/kotlin/io/github/mojira/arisa/ArisaMain.kt

```diff
@@ -124,7 +124,7 @@ private fun readLastRunTime(lastRun: List<String>): Instant {
 
 private fun readLastRun(lastRunFile: File): List<String> {
     return (if (lastRunFile.exists())
-        lastRunFile.readText()
+        lastRunFile.readText().trim()
```

Comment:
IIRC trim removes whitespace but not the last newline, specially as we receive a list of lines

Reply:
I believe whitespaces include newlines

Reply:
![image](https://user-images.githubusercontent.com/15277496/88968506-0d0a3200-d275-11ea-8b44-65d07cd5b7d8.png)

Yes, tested in [Kotlin playground](https://play.kotlinlang.org).